README.md

```diff
@@ -1,230 +1,207 @@
+c = \pm\sqrt{a^2 + b^2}
+{% endkatex %}
 ```
 
-## Migration Note
-
-### Migrating to 3.0.0
-
-* `mathjax` should be moved under `math` section in your `_config.yml`
-
-### Migrating to 2.1.0
+**MathJax**
 
-* `math_block` is removed
+```
+{% mathjax '{options}' %}
+content
+{% endmathjax %}
+```
 
-### Migrating to 2.0.0
+### Examples
 
-* `math_block` is deprecated (but still usable. This won't break your site)
-* `math` block is used to handle both inline and block math due to breaking changes made since Hexo 3.x. (This will break your site if you're using `math` block)
+```
+{% mathjax %}
+\frac{1}{x^2-1}
+{% endmathjax %}
+```
 
-Since Hexo 3.x, nunjucks is used as tag engine instead of swig. Syntax like this won't work any more:
+Override [front-matter](#per-article-configuration) and [global options](#global-options) for a particular content. Options must be specified in [JSON](https://en.wikipedia.org/wiki/JSON#Example) format.
 
-```markdown
-{% math \frac{|ax + by + c|}{\sqrt{a^{2}+b^{2}}} %}
 ```
-
-Instead, you should use open and close tag since this version:
-```markdown
-{% math %}\frac{|ax + by + c|}{\sqrt{a^{2}+b^{2}}} {% endmath %}
+{% mathjax '{ "conversion": { "em": 14 }, "tex": { "tags": "ams" }, "svg": { "exFactor": 0.03 } }' %}
+\frac{1}{x^2-1}
+{% endmathjax %}
 ```
 
-Change all your `math` tags accordingly.
+## Per-article configuration
 
-### Migrating to 1.0.6
+Override the global options via the front-matter of an article (post/page) basis.
 
-**!!!IMPORTANT!!!**
-
-Since 1.0.6, hexo-math uses a different approach to inject MathJax into your site. MathJax will be injected on-the-fly and on-demand. This means:
+``` yml
+---
+title: On the Electrodynamics of Moving Bodies
+categories: Physics
+date: 1905-06-30 12:00:00
+katex: false
+mathjax: false
+---
+```
 
-* **You are no longer required to run `hexo math install`**
-* MathJax will not be loaded on pages without math tags allowing faster loading speed.
+### Options
 
-If you have run `hexo math install` before, please **do run:**
+Disable math renderer in an article:
 
+``` yml
+---
+katex: false
+mathjax: false
+---
 ```
-$ hexo math
-```
-
-This will clean up previous installation. Or you can re-install your theme if for some reason this does not work.
 
+Override global options:
 
+``` yml
+---
+katex:
+  output: 'mathml'
+  felqn: true
+  minRuleThickness: 0.05
+  throwOnError: true
+mathjax:
+  conversion:
+    em: 14
+  tex:
+    tags: 'ams'
+  svg:
+    exFactor: 0.03
+---
+```
 
-### From 1.0.4 to 1.0.5 (Support Hexo 3.0)
+## Global Options
 
-Hexo 3.0 introduces multiple breaking changes. Versions before 1.0.5 won't work with it.
-The following changes are made since 1.0.5 to adapt the new 3.0 API:
+```yaml
+# _config.yml
+math:
+  katex:
+    css: 'https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css'
+    options:
+      throwOnError: false
+  mathjax:
+    css: 'https://cdn.jsdelivr.net/gh/hexojs/hexo-math/dist/style.css'
```

Comment:
use `https://cdn.jsdelivr.net/gh/curbengh/hexo-math@overhaul/dist/style.css` for testing.